batctl/Makefile

```diff
@@ -1,54 +1,47 @@
-#
-# Copyright (C) 2014 OpenWrt.org
-#
-# This is free software, licensed under the GNU General Public License v2.
-# See /LICENSE for more information.
-#
+# SPDX-License-Identifier: GPL-2.0
 
 include $(TOPDIR)/rules.mk
 
 PKG_NAME:=batctl
-
 PKG_VERSION:=2020.0
-PKG_RELEASE:=0
-PKG_HASH:=60efe9b148f66aa1b29110493244dc9f1f1d722e6d96969e4d4b2c0ab9278104
+PKG_RELEASE:=1
```

Comment:
I do not think the changes here require a bump of PKG_RELEASE, as they won't affect the content of the package if I read correctly.

Reply:
You misunderstand again. The PKG_RELEASE here is to follow the PKG_RELEASE numbering of the OpenWrt package feed.

Reply:
Ah, okay. However, in this case I'd still prefer to have the PKG_RELEASE bumped already with the PKG_VERSION increase.

Nevertheless, note that I have no power here, so I'm just suggesting what I think would be best, while the last word is left to the package maintainers. :-)